README.md

```diff
@@ -1 +1,165 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+> <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+> <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+> <li><a href="https://www.postgresql.org/">Postgresql</a></li>
+
+### Key Features <a name="key-features"></a>
+
+> - Add foods.
+> - List foods.
+> - Add recipes.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+## UML Diagram :
+
+<img src = 'recipe_diagram.png'>
+
+## To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Mac or PC
+- Install Ruby
+- Install Rails
+- Install Postgresql
+- Understanding of Ruby
+
+### Setup
+
+- Clone this repository to your desired folder:
+
+```sh
+ git clone https://github.com/rbhatt1999/recipe_app.git
+```
+
+- Navigate into the cloned folder
+
+```sh
+ cd recipe_app
+
+```
+
+- Install this project with:
+
+```sh
+ bundle install
+```
+
+- Set up the database
+  Change the username and password of your postgres account in `config/database.yml`
+
+```sh
+ rails db:create db:migrate
+```
+
+- Start the server
+
+```sh
+ rails s
+```
+
+- Run tests
+
+```sh
+ rails db:migrate RAILS_ENV=test
+```
+
+```sh
+ rspec spec
+```
+
+- if it shows error try using this command
+
+```sh
+ bundle exec rspec spec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors<a name="authors"></a>
+
+👤 **Hassan EL OUARDY**
+
+- GitHub: [@codehass](https://github.com/codehass)
+- Twitter: [@hassanelourdy](https://twitter.com/hassanelourdy)
+- LinkedIn: [Hassan El OUARDY](https://www.linkedin.com/in/hassan-el-ouardy-360b99169/)
+
+👤 **Rohit Bhatt**
+
+- GitHub: [@rbhatt1999](https://github.com/rbhatt1999)
+- Twitter: [@RohitBhatt](https://twitter.com/Rohit_Bhatt_)
+- LinkedIn: [Rohit Bhatt](https://www.linkedin.com/in/rohitbhatt-dev/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/rbhatt1999/recipe_app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please leave a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for providing us with reading materials that aided us to during the project development
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] 🚀 It seems that the LICENSE file has not been properly linked in the README. When I tried to open the LICENSE link in the README, it led me to a 💥broken link💥. I understand that things like this can happen and I believe it was an oversight on your part. No worries, it happens to the best of us 🤣. I kindly ask you to connect the LICENSE file to the README so it can be easily accessed by others. If you need any assistance or have any questions, please let me know 🤗.

Reply:
resolved